File: SortingJava/SelectionSort.java
/*Selection _ Sort */ 

class SelectionSort {
    static void selectionSort(int arr[],int size)
    {
        int i,j;

        for(i = 0; i < size-1; i++)
        {
            for(j = i+1; j < size; j++) 
            {
                if(arr[i] > arr[j])
                {
                    int t = arr[i];
                    arr[i] = arr[j];
                    arr[j] = t;
                }
            }
        }
    }
    
    
    public static void main(String[] args) {
        
        int[] arr = new int[] {180, 165, 150, 170, 145,156,175,134,127,113,106,170,153,161,183,194};
        int n = arr.length;
        
        System.out.println("Array in sorted order : ");
        selectionSort(arr,n);
        
        for(int i = 0; i < arr.length; i++)
            System.out.print(arr[i] + " ");
            
    }
    
}

File: SortingPyAlgo/SelectionSort.py
#Selection Sort Algorithm...

def selectionSort( arr , size )  : 

	for i in range( size-1 ) : 
		
		for j in range( i+1 , size ) :

			if ( arr[ i ] > arr[ j ] ):

				arr[i], arr[j] = arr[j], arr[i]


def printArr(arr, size) :
	print("[ ")
	for i in range(n):
		
		print(arr[i], end = " ")
	print(" ]")

if __name__=='__main__':

	arr = [ 32,45,26,13,67,75,17,96,23,15,52,27,87,35,24,56,78,65,86,48]	

	n = len(arr)
	print("Array_Len:\t",n)
	print("Non-Sorted-Arr :\t")
	printArr(arr, n)

	selectionSort(arr,n)

	print("\nArray in Sorted Order :\t")
	printArr(arr, n)
	print("\n")
	print(arr)





